### ISSUE TITLE
Error occurred when executing IPAdapterNoise:

### ISSUE THREAD

**Issue by Jamieharley** [cubiq/ComfyUI_IPAdapter_plus#590]
Error occurred when executing IPAdapterNoise:

cannot convert float infinity to integer

File "/workspace/ComfyUI/execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "/workspace/ComfyUI/execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "/workspace/ComfyUI/execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "/workspace/ComfyUI/custom_nodes/ComfyUI_IPAdapter_plus/IPAdapterPlus.py", line 1292, in make_noise
seed = int(torch.sum(image).item()) % 1000000007 # hash the image to get a seed, grants predictability

**Comment by cubiq** [comment]
what reference image are you sending?
